File: angle-between-hands-of-a-clock/angle-between-hands-of-a-clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double hours_angle=(hour%12)*30;
        double minutes_angle=minutes*6;
        hours_angle+=(double)minutes*30/60;
        double ans=Math.abs(hours_angle-minutes_angle);
        return Math.min(ans,360-ans);
    }
}

File: angle-between-hands-of-a-clock/angle-between-hands-of-a-clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double hour_angle=(hour%12)*30;
        double minutes_angle=minutes*6;
        hour_angle+=minutes_angle*30/360;
        double ans= abs(hour_angle-minutes_angle);
        return min(ans,360-ans);
    }
};